terraform-sentinel/index.json

```
@@ -0,0 +1,48 @@
+{
+  "title": "Sentinel Tutorial",
+  "description": "Writing & Testing Sentinel Policies in the Sentinel CLI and publishing to TFC",
+  "time": "30 minutes",
+  "icon": "fa-terraform",
+  "details": {
+    "intro": {
+        "text": "intro.md",
+        "code": "courseBase.sh",
+        "credits": "https://learn.hashicorp.com/terraform"
+    },
+        "assets": {
+      "host01": [
+
+        { "file": "restrict-s3-buckets.sentinel", "target": "~/" }
+      ]
+    },
+    "steps": [
+        {
+          "title": "Your first Sentinel Policy",
+          "text": "step1.md"
+        },
+        {
+          "title": "Adding additonal requirements",
+          "text": "step2.md"
+        },
+        {
+          "title": "Edit mock data",
+          "text": "step3.md"
+        },
+        {
+          "title": "Write passing and failing tests",
+          "text": "step4.md"
+        }
+      ],
+    "finish": {
+      "text": "finish.md",
+      "credits": "https://learn.hashicorp.com/terraform"
+    }
+  },
+  "environment": {
+    "uilayout": "editor-terminal",
```

```suggestion
    "uilayout": "editor-terminal",
    "hideHiddenFiles": true,
```
Include `hideHiddenFiles` to hide dotfiles in the editor.